movielens/setup.py

```diff
@@ -16,8 +16,10 @@
 
 NAME = 'movielens'
 VERSION = '1.0'
-TENSORFLOW_TRANSFORM = 'tensorflow-transform==0.1.10'
+SIX='six==1.10.0'
+TENSORFLOW='tensorflow==1.4.0rc'
```

Comment:
Why 1.4.0rc, I'd rather set to 1.3.0 since we are sure that's compatible with tensorflow-transform==0.3.1

Reply:
Ok checked 1.3 works and updated.